app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -40,7 +41,9 @@ define(function (require, exports, module) {
     commands: null,
 
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
-      sendChangePasswordNotice: true
+      sendAfterSignInConfirmationPollNotice: false,
+      sendAfterSignUpConfirmationPollNotice: false,
```

Comment:
No, this is the semantics of the content server. "Confirm" occurs in the tab the user initiated the action. "verify" occurs in the verification tab.